src/frontend/packages/lib_components/src/hooks/stores/useResponsive/index.spec.tsx

```diff
@@ -3,10 +3,9 @@ import { renderHook, WrapperComponent } from '@testing-library/react-hooks';
 import { ResponsiveContext } from 'grommet';
 import { appendUtilsElement, render } from 'lib-tests';
 import React, { Fragment } from 'react';
+import { getFullThemeExtend } from 'standalone_site/src/styles/theme.extend';
```

Comment:
What is needed here are the breakpoints in `standalone_site/src/styles/theme.extend`, I think you should export the breakpoints from `standalone_site` and put them in `lib-common/theme` in a separate constante, then you can deepMerge them in the `standalone_site`, and do the same here.

Reply:
Please tell me if I done it right : https://github.com/openfun/marsha/pull/2121/commits/6c790b6f309f4a49c8fae955d599fa0beca843c6

Reply:
Looks good 🤘